app/src/main/java/org/openmrs/eip/app/management/entity/DebeziumEvent.java

```diff
@@ -0,0 +1,50 @@
+package org.openmrs.eip.app.management.entity;
+
+import javax.persistence.AttributeOverride;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import org.openmrs.eip.component.entity.Event;
+
+@Entity
+@Table(name = "debezium_event_queue")
+public class DebeziumEvent extends AbstractEntity{
+	private static final long serialVersionUID = -1884382844867650350L;
+	
+    @Embedded
+    @AttributeOverride(name = "identifier", column = @Column(updatable = false))
+    @AttributeOverride(name = "primaryKeyId", column = @Column(name = "primary_key_id", nullable = false, updatable = false))
+    @AttributeOverride(name = "tableName", column = @Column(name = "table_name", nullable = false, updatable = false, length = 100))
+    @AttributeOverride(name = "operation", column = @Column(nullable = false, updatable = false, length = 1))
+    @AttributeOverride(name = "snapshot", column = @Column(nullable = false, updatable = false))
+    private Event event;
+	
+    private Long eventTimeStamp;
+    
+    public DebeziumEvent() {
+    	this.event = new Event();
+    }
+    
+    public void setEvent(Event event) {
+		this.event = event;
+	}
+    
+    public Event getEvent() {
+		return event;
+	}
+    
+	public Long getEventTimeStamp() {
+		return eventTimeStamp;
+	}
+
+	public void setEventTimeStamp(Long eventTimeStamp) {
+		this.eventTimeStamp = eventTimeStamp;
+	}
+     
+	@Override
+	public String toString() {
+		return event.toString();
```

Comment:
I think you need to log the important field values like we do in other entity classes e.g. PK, identifier, table, operation, snapshot